source/parser.hera

```diff
@@ -1166,6 +1184,15 @@ Arrow
     return { $loc, token: $1}
 
 ExplicitBlock
+  __ OpenBrace __ CloseBrace ->
```

Comment:
I've been starting to replace the loose leading `__` with `( Samedent / _ )?`. The `__` in between open and close in fine.

Not something that needs to be done in this PR just something I wanted to share as we move towards more strict/robust indenting.